main.go

```diff
@@ -39,12 +39,17 @@ func main() {
 		config: config,
 		bot:    bot,
 	}
+
+	if err = loadNotificationSettings(); err != nil {
+		log.Println(err)
```

Comment:
Since this is a log message (not a returned error, which will be part of another message) it would be a good idea to say what triggered the error. Something like `log.Printf("Error loading notifications: %v", err)`